src/kaggle/api/kaggle_api_extended.py

```diff
@@ -146,7 +151,7 @@ class ResumableFileUpload(object):
   # A resumable upload must be completed within a week of being initiated
   RESUMABLE_UPLOAD_EXPIRY_SECONDS = 6 * 24 * 3600
 
-  def __init__(self, path, start_blob_upload_request, context):
+  def __init__(self, path: str, start_blob_upload_request: ApiStartBlobUploadRequest, context: ResumableUploadContext) -> None:

```

Comment:
not sure why sometimes it decides to `"` ResumableUploadContext and not other times.  I guess if it works 🤷

Reply:
That's because, until the class is fully defined, it's name is not known. The current practice is to just return the name as a string. That's in flux, and in 4.0 the name should be available.